templates/execute-query-send-result.json

```diff
@@ -0,0 +1,305 @@
+{
+    "id": "execute-query-send-result",
+    "title": "execute-query-send-result",
+    "description": "Executes and sends you the query results on recurrence",
```

Comment:
I don't think we address customers as "you", maybe something like "Executes and send query result via e-mail on schedule."